dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/validator/EnrollmentAttributeValidatorTest.java

```diff
@@ -176,8 +176,9 @@ void shouldFailValidationWhenValueIsNullAndAttributeIsMandatory()
 
         validator.validate( reporter, bundle, enrollment );
 
-        assertThat( reporter.getErrors(), hasSize( 1 ) );
-        hasTrackerError( reporter, ValidationCode.E1076, TrackerType.ENROLLMENT, enrollment.getUid() );
+        assertAll(
```

Comment:
If this is a very common pattern for an assertion, would it make sense to create something like `assertHasOnlyError` to wrap this logic?

Reply:
Often enough, yes we could create it. I just wonder whether we should even assert on the error size at all 🤔